my-app/src/App.js

```diff
@@ -0,0 +1,57 @@
+//// this is your original code before you tried to do this and you broke it. good job james
+import React, { Component } from 'react';
+import './App.css';
+import Header from './Header.js';
+import PokeList from "./PokeList.js";
+import SearchOptions from './SearchOptions.js'
+import request from 'superagent'; 
+import getPokemon from './getPokemon.js';
```

Comment:
I don't see anything in your `getPokemon.js` file so this will fail when you try to run a search.